rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -1926,6 +1929,11 @@ def test_upgrade_db_39_to_40(user_data_dir):  # pylint: disable=unused-argument
     assert cursor.execute('SELECT type, seq FROM action_type').fetchall() == [
         ('A', 1), ('B', 2), ('C', 3),
     ]
+    # check that we have a defined accounting rule for aidrops comming from ledger actions
```

Comment:
can you also check that the value of the rule agrees with the value of the saved `taxable_ledger_actions` ?